shardy/dialect/sdy/ir/attrs.td

```diff
@@ -804,10 +804,18 @@ def Sdy_OpShardingRule : AttrDef<Sdy_Dialect, "OpShardingRule"> {
 def Sdy_AxisRefList : ArrayOfAttr<Sdy_Dialect, "AxisRefList",
                                  "axis_ref_list", "AxisRefAttr"> {
   let assemblyFormat = "`{` (`}`) : ($value^ `` `}`)?";
+  let summary = "List of axis refs";
+  let parameters = (ins
```

Comment:
Oh, similar to ManualAxes, we should remove the parameters and just leave summary. ditto for Sdy_ListOfAxisRefLists